acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/tls/support/MockBalanceReporter.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.dsl.tls.support;
+
+import tech.pegasys.web3signer.core.service.jsonrpc.JsonRpcRequest;
+import tech.pegasys.web3signer.core.service.jsonrpc.JsonRpcRequestHandler;
+import tech.pegasys.web3signer.core.service.jsonrpc.handlers.HttpResponseFactory;
+
+import io.vertx.ext.web.RoutingContext;
+
+public class MockBalanceReporter implements JsonRpcRequestHandler {
```

Comment:
Think this should be moved out of tls package...to the shared utils maybe?

Reply:
done